### ISSUE TITLE
wrong use of nn.CrossEntropyLoss in train_mobilenetV2.ipynb?

### ISSUE THREAD

**Issue by amrta-coder** [Miura55/deeplearning-face-mask-detection#1]
Since nn.CrossEntropyLoss is used, I am afraid that using nn.LogSoftmax in the model will make it a conflict.
If I make misunderstood, please let me know.
